api/custom-metrics-api.openapi.yaml

```diff
@@ -86,19 +82,7 @@ components:
           type: string
           description: An optional unit for the metric, for display purposes only.
           example: jobs
-    ErrorResponse:
-      type: object
-      properties:
-        code:
-          type: string
-          description: The textual error code.
-        message:
-          type: string
-          description: The message describing the cause of the error.
   securitySchemes:
-    mtls:
-      type: mutualTLS
-      description: Use the Instance Identity Credentials provided by the Cloud Foundry runtime
     basicAuthentication:
       type: http
-      scheme: basic
```

Comment:
It seems your change removes the mtls security scheme completely, I would prefer to keep it documented.

Reply:
Unfortunately some tools still don't work with OpenAPI-3.1.0. This in spite of being not that new already. :(

The cost here is that if we include it, we need to exclude the whole custom-metrics-spec from the openapi-validation in the workflow `openapi-specs-check. Furthermore we can not easily generate the required go-code. The reason is that we can not generate anymore bundled specs which are required by our not so perfectly spec-compliant golang-code-generator.

Reply:
We are able to use the forked swagger-cli tool. Skipping security {"at": "custom-metrics-api.openapi.bundled.yaml:77:11", "reason_error": "mutualTLS security not implemented"} for the documentation process and has no effect on the generated code